<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaoWxPay</class>
 <widget class="QMainWindow" name="TaoWxPay">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>271</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vLayOut"/>
   </widget>
   <widget class="QLabel" name="lblMsg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>560</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>lblMsg</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblWxPayLogo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>260</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>lblWxPayLogo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblWxPayExplainText">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>260</width>
      <height>86</height>
     </rect>
    </property>
    <property name="text">
     <string>lblWxPayExplainText</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
